examples/hundred-tasks/src/main/java/io/littlehorse/examples/MyWorker.java

```diff
@@ -11,7 +11,11 @@ public String task1() {
 
     @LHTaskMethod("task-2")
     public String task2() {
-        return "task-2";
+        StringBuilder out = new StringBuilder();
+        for (int i = 0; i < 100; i++) {
+            out.append("asdfasdfasdfasdfjwth0238ahs9213t8hd");
```

Comment:
is this going to be removed?

Reply:
Good catch, will remove. I added it so that we could build up state faster so that restoration times would take longer, so that I could actually test everything.

But I don't think it needs to be in the examples :+1: